prefetch.bs

```diff
@@ -225,19 +228,40 @@ A <dfn export>cross-origin prefetch IP anonymization policy</dfn> has an <dfn ex
   1. Return false.
 </div>
 
+A <dfn export>prefetch controller</dfn> is a [=struct=] which allows managing a prefetch operation. It has the following [=items=]:
+* <dfn for="prefetch controller">state</dfn> (default `"ongoing"`), which is `"ongoing"` or `"aborted"`
```

Comment:
This is only ever set, never consulted. So I think it can be removed?

Reply:
Its replacement is consulted.